src/utils/frappeRequest.js

```diff
@@ -90,14 +91,6 @@ export function frappeRequest(options) {
             : ['Internal Server Error']
         }
 
-        if (options.onError) {
```

Comment:
@netchampfaris 
This was good for catching all the errors at the request level. 
For eg. creating a toast if any request failed or routing to `/login` if unauthorized